libraries/AP_Notify/mRoLED_I2C.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+  mRoLED I2C driver
+*/
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* LED driver for mRo module */
+
+#include "mRoLED_I2C.h"
+
+#include <utility>
+
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+#define mRo_LED_I2C_ADDR     0x38 // default I2C bus address of NCP5623
+                                  // Note that it has no sub-address nor read-back register
+#define CMD_SHUTDOWN            0
+#define CMD_MAX_CURRENT      0x38
+#define PH2_I2C_BUS_INTERNAL    1 // I2C1 on GPS1 connector, 3.3V
+#define PH2_I2C_BUS_EXTERNAL    0 // I2C2 connector or I2C2 on GPS2, 3.3 - 5 V
+
+#define mRoDebug true
+
+bool mRoLED_I2C::hw_init()
+{
+    // first look for led on external bus
+/* cubirootk
+    _dev = std::move(hal.i2c_mgr->get_device(PH2_I2C_BUS_INTERNAL, mRo_LED_I2C_ADDR));
```

Comment:
I think that it is better to delete the commented out source, if it is unnecessary.